aspnetcore/fundamentals/url-rewriting/samples/2.x/SampleApp/README.md

```diff
@@ -21,25 +21,30 @@ When running the sample, a response will be served that shows the rewritten or r
 * `Add(RedirectXMLRequests)`
   - Success status code: 301 (Moved Permanently)
   - Example (redirect): **/file.xml** to **/xmlfiles/file.xml**
+* `Add(RewriteTXTRequests)`
```

Comment:
nit: I'd consider changing "TXT" to something clearer. I wasn't sure at first what you meant by it when I read it as part RewriteTXTRequests. Just looks a little odd to me, but maybe clear to others